### ISSUE TITLE
Initialization Time

### ISSUE THREAD

**Issue by brian-eng** [adafruit/Adafruit_CircuitPython_MLX90393#42]
As part of the sensor initialization a reset is performed. This function has a call to time.sleep(2) between the EXIT and RESET commands. Why is there such a long wait there? This probably is not much of an issue when only using one sensor but when trying to use multiple sensors this delay really adds up quickly.

According to the datasheet 

> It is recommended to perform an ‘EX’ command and to wait 1ms, before issuing a ‘RT’ command. After the RT command, it is recommended to wait 1.5ms for the start-up sequence to complete.

Maybe that was meant as 2 ms? However, there is no delay after the reset itself.

**Comment by caternuson** [comment]
Maybe 2ms was the intention, based on the POR timing from the datasheet?
![image](https://github.com/user-attachments/assets/32c8f325-e648-4b6c-93ba-d1df462fe971)

Pretty common to be conservative and take the max value and round it up, which would support the idea of this intending to be a 2ms wait.

Also, Arduino's ms based `delay()` sometimes gets mixed up with Python's seconds based `time.sleep()` - for when code is ported, or if by someone used to writing Arduino code, etc. 

But if that was the intention, the delay should be after sending the reset command. It's not even related to the sending of the EXIT command, since the delay has been there before that was even added (v1 release):
https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/blob/f9a1e2ec316e752ee92aefd9dbc9f9dc65bea98b/adafruit_mlx90393.py#L225-L233


**Have you tried removing it?**

**Comment by brian-eng** [comment]
> **Have you tried removing it?**

Yes, I have commented out the 2 second delay and it works fine on my end. However, I'm currently testing using Blinka on a RPi 5, so not exactly a standard CircuitPython setup.

I've also put a `time.sleep(0.001)` after the EXIT and `time.sleep(0.002)` ms after the RESET and that works as well.

**Comment by caternuson** [comment]
Thanks. That also generally verifies that 2 second delay isn't really needed. Based on the datasheet, seems like a basic reset would be:
* send EX  (exit) command
* send RT (reset) command
* wait at least 1.5ms

So why is the 2s delay there? Don't know. You've tested without and it works. The Arduino library has no 2 second delay.

A PR to remove it seems fine.
